vendor/priors/gwt.js

```diff
@@ -319,10 +319,46 @@ function generateCrypto_box_keypair(publicKey, secretKey) {
     return publicKey;
 }
 
+var tryProofOfWork = function(counter, difficulty, combined, future) {
+    combined[0] = counter;
+    combined[1] = (counter >> 8);
+    combined[2] = (counter >> 16);
+    combined[3] = (counter >> 24);
+    combined[4] = (counter / 4294967296);
+    combined[5] = ((counter / 4294967296) >> 8);
```

Comment:
maybe just add a comment so it is clear what is going on here eg equivalent to counter >> 32 in java

Reply:
done